website/docs/configuration.md

```diff
@@ -269,6 +269,24 @@ Lua:
 vim.g.neovide_transparency = 0.8
 ```
 
+#### Show Border (Currently macOS only)
+
+VimScript:
+
+```vim
+let g:neovide_show_border = v:true
+```
+
+Lua:
+
+```lua
+vim.g.neovide_show_border = true
+```
+
+Draw a grey border around the window, as seen on most native macOS apps.
```

Comment:
just describe what it does and the conditions it should apply.

for example the opacity rule.

something like:

```suggestion
Draw a grey border around opaque windows only.
```